packages/kit/test/apps/basics/src/hooks.server.js

```diff
@@ -44,6 +44,17 @@ export const handle = sequence(
 		event.locals.answer = 42;
 		return resolve(event);
 	},
+	({ event, resolve }) => {
+		if (
+			event.request.url.includes('__data.json') &&
```

Comment:
this would prevent contrived-but-allowed things like `/__data.json/wheee`, no? Don't we just want to check the end of the pathname?

Reply:
This is just part of a test, not the code that users are going to run.

Reply:
i definitely knew that and wasn't just being absent-minded